Sources/web3swift/Utils/ENS/ENSRegistry.swift

```diff
@@ -65,6 +65,16 @@ public extension ENS {
             return Resolver(web3: self.web3, resolverContractAddress: resolverAddress)
         }
 
+        let hexCharacters: [Character] = Array("0123456789abcdef")
```

Comment:
Let's not mix functions and variables. Move it up and I'd consider making it static.